django_saas_email/models.py

```diff
@@ -257,6 +300,8 @@ class AbstractMail(models.Model):
         editable=False,
     )
 
+    attachments = models.ManyToManyField(Attachment, null=True, blank=True)
```

Comment:
This field should also use `through=TemplateAttachment` I think.

Reply:
On second thought, please ignore this suggestion.

Reply:
Probably this should be named `selected_attachments` to make clearer, that this is about the uploaded attachments, which have been uploaded before. 

LATER we might add `uploaded_attachments` with link to `UploadedMailAttachment` so it would be good, to have this with clear naming.